org.mwc.debrief.core/intro_content/debrief_tutorials.xml

```diff
@@ -4,29 +4,15 @@
 		alt-style="css/overview.properties" style="css/intro.css" name="Debrief bits" path="tutorials/@">
 		<group label="Finding your sea legs in DebriefNG" style-id="content-group"
 			id="orgmwcdebriefcore-introLink-group">
-			<link style-id="content-link" label="A Debrief walkthrough"
-				url="http://org.eclipse.ui.intro/showStandby?partId=org.eclipse.platform.cheatsheet&amp;input=org.mwc.debrief.help.started.tutorial"
+			<link style-id="content-link" label="Debrief tutorial"
+				url="http://org.eclipse.ui.intro/runAction?pluginId=org.mwc.debrief.core&#38;class=org.mwc.debrief.core.actions.OpenDebriefTutorialAction&#38;file=DebriefNGTutorial.pdf"

```

Comment:
I open pdf with OS's standard command and if it doesn't succeed open url in browser.
It is better way because we only depends on browser's plugins/addons.
We also have to find the location of pdf file (Debrief install location) what is easier in an action (not sure if it is possible to add install location to url).
In any case it is the simple action.